cimi/src/com/sixsq/slipstream/ssclj/resources/version.clj

```diff
@@ -0,0 +1,74 @@
+(ns com.sixsq.slipstream.ssclj.resources.version
```

Comment:
I'd prefer to see a more specific name for this; "version" is too generic.  Maybe "ModuleVersion", "ModuleInstance" or "ModuleContent".